azurerm/internal/services/compute/shared_image_version_data_source.go

```diff
@@ -133,6 +138,12 @@ func dataSourceArmSharedImageVersionRead(d *schema.ResourceData, meta interface{
 				osDiskSnapShotID = *profile.OsDiskImage.Source.ID
 			}
 			d.Set("os_disk_snapshot_id", osDiskSnapShotID)
+
+			osDiskImageSize := -1
+			if profile.OsDiskImage != nil && profile.OsDiskImage.SizeInGB != nil {
+				osDiskImageSize = int(*profile.OsDiskImage.SizeInGB)
+			}
+			d.Set("os_disk_image_size", osDiskImageSize)
```

Comment:
since this is in GB - can we make this:

```suggestion
			d.Set("os_disk_image_size_gb", osDiskImageSize)
```